### PR TITLE
Add missing options to docs and improve naming

### LINKED ISSUE #1272: "Checkout the examples" link in README.md does not link to examples
I'm not certain where it should link to, perhaps here: https://github.com/jitsi/lib-jitsi-meet/tree/master/doc/example ?

### PR DESCRIPTION
**Pull request by sapkra**
Closes #1272

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -232,13 +237,52 @@ This objects represents the server connection. You can create new ```JitsiConnec
...
+        - forceJVB121Ratio - "Math.random() < forceJVB121Ratio" will determine whether a 2 people conference should be moved to the JVB instead of P2P. The decision is made on the responder side, after ICE succeeds on the P2P connection.
+        - hiddenDomain
+        - startAudioMuted
+        - startVideoMuted
+        - enableLayerSuspension - if set to 'true', we will cap the video send bitrate when we are told we have not been selected by any endpoints (and therefore the non-thumbnail streams are not in use).
+        - deploymentInfo
+            - shard
+            - userRegion
+        - p2p - Peer to peer related options
+            - enabled - enables or disable peer-to-peer connection, if disabled all conferences or going through the videobridge
```
Comment (jallamsetty1) [inline doc/API.md:265]: Modification - if disabled all media will be routed through the Jitsi Videobridge.

File: doc/API.md [hunk 2]
```diff
@@ -232,13 +237,52 @@ This objects represents the server connection. You can create new ```JitsiConnec
...
+            - disableH264
```
Comment (jallamsetty1) [inline doc/API.md:266]: `disableH264` and `preferH264` are deprecated options. Can you please mark them deprecated and add `preferredCodec` and `disabledCodec` which are the new options ?

File: doc/API.md [hunk 3]
```diff
@@ -232,13 +237,52 @@ This objects represents the server connection. You can create new ```JitsiConnec
...
+            - getStatsInterval
+            - analyticsInterval
+            - stunServers
+        - e2eping
+            - pingInterval
+        - abTesting - A/B testing related options
+            - enableSuspendVideoTest
+        - testing
+            - capScreenshareBitrate
+            - enableFirefoxSimulcast
```
Comment (jallamsetty1) [inline doc/API.md:282]: enableFirefoxSimulcast is no longer supported. Simulcast is enabled by default on FF now. The only way to disable it by using `disableSimulcast` which will disable it across all endpoints.

**Review by jallamsetty1 — COMMENTED** [review]
Thanks @sapkra for updating the doc. I left a few comments.

File: doc/API.md (hunk 2 above)
Comment (sapkra) [reply doc/API.md:266]: `backToP2PDelay` was also missing. Are there more configs which were recently added but still missing in the docs?

Comment (jallamsetty1) [reply doc/API.md:266]: cool! I don't think I see any other options that are missing or no longer valid.

**Comment by sapkra** [comment]
I applied everything and added a new commit which changed the formatting a bit so that everything looks the same and is more simplified.

btw. there are also some other docs related PR which can easily be reviewed and merged. It would be great to clean the PRs up a bit:
https://github.com/jitsi/lib-jitsi-meet/pull/1323
https://github.com/jitsi/lib-jitsi-meet/pull/1327
https://github.com/jitsi/lib-jitsi-meet/pull/1273
https://github.com/jitsi/lib-jitsi-meet/pull/1184
https://github.com/jitsi/lib-jitsi-meet/pull/1074
https://github.com/jitsi/lib-jitsi-meet/pull/1067
https://github.com/jitsi/lib-jitsi-meet/pull/1060

And this could be easily merged & it's already approved:
https://github.com/jitsi/lib-jitsi-meet/pull/1019

Would very appreciate it if you could find the time for it. I don't have the permissions.

**Comment by jallamsetty1** [comment]
> I applied everything and added a new commit which changed the formatting a bit so that everything looks the same and is more simplified.
> 
> btw. there are also some other docs related PRs which can easily be reviewed/merged. It would be great to clean the PRs up a bit:
> #1323
> #1327
> #1273
> #1184
> #1074
> #1067
> #1060
> 
> And this could be easily merged & it's already approved:
> #1019
> 
> Would very appreciate it if you could find the time for it. I don't have the permissions.

Sure, I will start going through them.

**Review by jallamsetty1 — APPROVED** [review]

**Comment by WesUnwin** [comment]
Hi, I'm the author of PR 1067, my PR fixes a definite typo, was made close to a year ago.

I accepted the contribution agreement at the time immediately.

I wanted to ask if your generally accepting contributions from people like me?

Can you also quickly confirm if you even still need my PR? I feel like my chances of being able to contribute are now 0%.

The other PR I owned 1059, was just closed with no explanation. :(

**Comment by sapkra** [comment]
I already asked for open source community managers which are able to review PRs multiple times but it doesn't seem to happen anything. The review politics of 8x8 in general are very bad and can be improved by a lot. I already discussed this topic with @saghul

btw. even though I'm a member, I'm just part of the community and not working for 8x8.

The docs related PRs were closed because the docs have been moved to the [handbook](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-ljm-api) repository.

**Pull request merged**